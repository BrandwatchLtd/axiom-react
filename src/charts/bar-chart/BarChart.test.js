import React from 'react';
import renderer from 'react-test-renderer';
import { BarChart } from 'bw-axiom';

function getComponent() {
  return renderer.create(
    <BarChart height={ 256 }/>
  );
}

describe('BarChart', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
