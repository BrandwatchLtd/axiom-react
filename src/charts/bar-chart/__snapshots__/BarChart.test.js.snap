// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarChart renders with defaultProps 1`] = `
<div
  className="ax-bar-chart"
  style={
    Object {
      "height": 256,
    }
  }
/>
`;
